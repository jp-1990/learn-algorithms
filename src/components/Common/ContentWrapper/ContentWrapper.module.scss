.wrapper {
  width: 100%;
  height: auto;
  background-color: $background;
  @include flexCenter(center, center);
}
