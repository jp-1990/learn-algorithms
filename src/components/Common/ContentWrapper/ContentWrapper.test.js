import React from "react"
import renderer from "react-test-renderer"

import ContentWrapper from "./ContentWrapper"

describe("<ContentWrapper />", () => {
  const defaultProps = <div>Children</div>
  const wrapper = renderer.create(<ContentWrapper children={defaultProps} />)

  test("render", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
