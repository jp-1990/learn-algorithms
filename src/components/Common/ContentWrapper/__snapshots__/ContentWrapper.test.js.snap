// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentWrapper /> render 1`] = `
<div
  className="wrapper"
>
  <div>
    Children
  </div>
</div>
`;
